src/components/pages/login/LoginForm.js

```diff
@@ -35,7 +50,11 @@ export default function LoginForm() {
             onChange={handleChange}
             required 
         />
-        <button type="submit">Accédez à votre espace</button>
+        <button 
+            onClick={() => !account ? setGoToOrderPage(false) : setGoToOrderPage(true)} 
```

Comment:
on peut simplifier le code ici avec ceci :

```suggestion
            onClick={() => hasAccount &&  setGoToOrderPage(true)} 
```